### DISCUSSION TITLE
My GNOME Tweaks - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#32] [General]
# My GNOME Tweaks - Alan Pope's blog

One of the neat things about GNOME Shell is that it’s pretty tweakable - to some degree - to customise it to a user’s preferences. I know some people use GNOME Shell stock experience. I don’t. I have previously written about some of my must-have extensions and add-ons. This supplements that with what I do to further tweak my (currently) Ubuntu 20.10 system to my liking.
Note: These are the settings I configure on my computer that I use all day every day.

https://popey.com/blog/2021/01/my-gnome-tweaks/

**Comment by davidjao** [comment]
The use case for having only the current monitor move when you switch workspaces is: I am giving a presentation. The projector is configured as a second monitor. I would very much like to be able to switch workspaces on my built in display on my laptop, so that I can do things such as switch between sets of notes, while keeping the presentation image static.

**Reply by davidjao** [reply]
Oh and by the way

> Being able to switch workspaces quickly with a deft swipe of the hand on the trackpad is great. Also, being able to “peek” across one workspace to another, by half-sliding across the trackpad is just 😍 - I wish we had that. I might use trackpads then!

we have that now. Three finger swipe works out of the box on Ubuntu 22.04, and you can peek across workspaces. The catch is, it only works in Wayland.
